pkgs/applications/misc/lswt/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland }:
+
+stdenv.mkDerivation rec {
+  pname = "lswt";
+  version = "1.0.4";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Orwa7sV56AeznEcq/Xj5qj4PALMxq0CI+ZnXuY4JYE0=";
+  };
+
+  makeFlags = [
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
+  ];
+
+  meta = with lib; {
+    description = "A command that lists Wayland toplevels";
+    homepage = "https://git.sr.ht/~leon_plickat/lswt";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ edrex ];
+    platforms = platforms.linux;
```

Comment:
are we sure this only works on linux, iirc wayland ran on bsd? happy to wait for somebody to ask/confirm bsd support.

Reply:
i was just copying platform from other wayland stuff (river, sway). I can't test. Maybe at some point a *bsd user could go through bulk-update supported wayland tools?